admin/brand.php

```diff
@@ -4,12 +4,12 @@
 
     if($_SERVER['REQUEST_METHOD'] == 'POST') {
         $error_msg = [];
-        if(empty($_POST['Brand_ID'])) {
-                $error_msg[0] = 'Brand ID field cannot be empty.';
+            if(empty($_POST['Brand_ID'])) {
+                //Ele what do we do
```

Comment:
If there is no Brand ID, then we probably don't want to continue validating, right? In your <form> action, are you passing the brand id as a parameter there as well?